tests/yast2_gui/yast2_control_center.pm

```diff
@@ -273,6 +291,12 @@ sub run {
     if (is_sle && sle_version_at_least '15') {
         #see bsc#1062331, sound is not added to the yast2 pattern
         ensure_installed('in yast2-sound');
+        #kdump is disabled by default, so ensure that it's installed
+        ensure_installed('in yast2-kdump');
+        record_soft_failure 'bsc#1059569';
```

Comment:
Would it work to check for the error condition first? E.g. with multi-tag `assert_screen` and `match_has_tag`?

Reply:
Which error condition? We can verify that package is not installed, then soft-fail and install, or see that module is not there -> install -> restart control center, but that looks like an overhead for this case. We can also just fail on the bug and see which decision is made.

Reply:
ok, fine then